app/views/records/_quick_filters.html.haml

```diff
@@ -6,19 +6,18 @@
   \#
   \# Supplejack was created by DigitalNZ at the National Library of NZ and the Department of Internal Affairs.
   \# http://digitalnz.org/supplejack
-.quick-filters.active
+#panel1-1.quick-filters.content.active
   - filter_facets.each_with_index do |facet, index|
-    %section
-      %st> 
-        = facet[0].capitalize.gsub('_', ' ')
+    %section.large-4.columns
+      = facet[0].capitalize.gsub('_', ' ')
       %ul.columns
         - facet[1].each_with_index do |filter, i|
           - if i < 5
+
             %li
               = content_tag :a,                                                                                                        |
-                "#{filter[0]} <span class='count'>#{number_with_precision(filter[1], delimiter: ',', precision: 0)}</span>".html_safe, |
+                "#{date_parser_for(filter[0])} <span class='count'>#{number_with_precision(filter[1], delimiter: ',', precision: 0)}</span>".html_safe, |
                 class: filter_class(filter[0]),                                                                                        |
                 data: {facet: facet[0], filter: filter[0]}                                                                             |
         %li
-          %a.more{:href => "#panel1-#{index + 2}"} More +
-      %hr    
+          %a.more{:href => "#panel1-#{index + 2}"} More +  
```

Comment:
Hash attribute should start with one space after the opening brace<br>Hash attribute should end with one space before the closing brace<br>Line contains trailing whitespace